packages/teleterm/src/sharedProcess/ptyHost/ptyProcess.ts

```diff
@@ -47,7 +47,7 @@ export class PtyProcess extends EventEmitter implements IPtyProcess {
       name: 'xterm-color',
       // HOME should be always defined. But just in case it isn't let's use the cwd from process.
       // https://unix.stackexchange.com/questions/123858
-      cwd: this.options.cwd || this.options.env['HOME'] || process.cwd(),
+      cwd: this.options.cwd || getDefaultCwd(),
```

Comment:
Let's pass `this.options.env` to `getDefaultCwd` and use it there for `USERPROFILE`/`HOME` instead of `process.env`. I assume we pass that env along for a reason.

Reply:
Correct, it should be done this way